vault/expiration.go

```diff
@@ -884,8 +889,10 @@ func (m *ExpirationManager) Stop() error {
 	m.uniquePolicies = make(map[string][]string)
 	m.irrevocable.Range(func(key, _ interface{}) bool {
 		m.irrevocable.Delete(key)
+		m.irrevocableLeaseCount--
```

Comment:
I'm not sure that there's a point to this decrement if we are setting it back to 0 anyway on line 895

Reply:
2558e12